src/index.js

```diff
@@ -41,18 +41,29 @@ export default class Form extends React.Component {
   floatingLabelFocusStyle = this.props.focusStyle || {};
 
   handleEmojiText = (input) => {
-    var emoji = new EmojiConvertor();
-    return emoji.replace_colons(input);
+    if(_.isString(input)) {
+      const emoji = new EmojiConvertor();
+      return emoji.replace_colons(input);
+      }
+     else return input;
   }
 
   handleChange = (field, value) => {
-    const valueWithEmoji = this.handleEmojiText(value);
+    const { fields } = this.props;
+    const wholeField = fields.find(({name, key}) => name === field || key === field)
```

Comment:
What does this wholeField means?

Reply:
`field` actually is no the whole field, it is referring to the field name only, it's just a string. `wholeField` is an object containing the whole field including properties such as name, type, key, emoji, items, ...